dags/public-health/Code55.py

```diff
@@ -0,0 +1,153 @@
+    df = pd.DataFrame(cleandict)
+
+    # Deduplicating DataFrame
+    df["CompositeID"] = (
+        df["Number"].astype(str) + "-" + df["AssociatedServiceRequestNumber"]
+    )
+    df.drop_duplicates(subset="CompositeID", inplace=True)
+
+    # Assigning dataset
+    df["Stage"] = stage
+
+    # Adjusting datetimes from unix timestamp to readable date
+    datelist = [
+        "DateCreated",
+        "DateApproved",
+        "ExpirationDate",
+        "DateCompleted",
+        "CleanupDate",
+    ]
+    for date in datelist:
+        df[date] = pd.to_datetime(df[date], unit="ms")
+    return df
+
+
+def update_code_55(**kwargs):
+    # Collecting each dataset
+    code_55_closed_last_90 = get_code_55_df(0, "Closed")
+    code_55_pending_schedule = get_code_55_df(1, "Pending Scheduled")
+    code_55_scheduled = get_code_55_df(2, "Scheduled")
+
+    # Prioritizing data where Closed Last 90 > Scheduled > Pending Schedule
+    code_55_pending_schedule = code_55_pending_schedule.loc[
+        ~(
+            code_55_pending_schedule["CompositeID"].isin(
+                code_55_closed_last_90["CompositeID"]
+            )
+        )
+        & ~(
+            code_55_pending_schedule["CompositeID"].isin(
+                code_55_scheduled["CompositeID"]
+            )
+        )
+    ]
+    code_55_scheduled = code_55_scheduled.loc[
+        ~code_55_scheduled["CompositeID"].isin(code_55_closed_last_90["CompositeID"])
+    ]
+
+    # Merging all data into one dataframe
+    df = pd.concat(
+        [code_55_closed_last_90, code_55_pending_schedule, code_55_scheduled]
+    )
+
+    # Creating table if does not exist
+    engine = PostgresHook.get_hook("postgres_default").get_sqlalchemy_engine()
+    create_table_statement = """CREATE TABLE IF NOT EXISTS "public-health"."code55s" (
+    index BIGINT,
+    "Id" BIGINT,
+    "GlobalID" TEXT,
+    "Number" TEXT,
+    "AssociatedServiceRequestNumber" TEXT,
+    "DateCreated" TIMESTAMP WITHOUT TIME ZONE,
+    "DateApproved" TIMESTAMP WITHOUT TIME ZONE,
+    "ExpirationDate" TIMESTAMP WITHOUT TIME ZONE,
+    "Status" BIGINT,
+    "Address" TEXT,
+    "City" TEXT,
+    "ZipCode" TEXT,
+    "CrossStreet" TEXT,
+    "CouncilDistrict" TEXT,
+    "APREC" TEXT,
+    "LocationComments" TEXT,
+    "SubmittedBy" TEXT,
+    "ReportingPerson" TEXT,
+    "RPContactNo" TEXT,
+    "Details" TEXT,
+    "HEAssessmentBy" TEXT,
+    "AssessmentContactNo" TEXT,
+    "HEAssessmentDetails" TEXT,
+    "AssessmentLocationDescription" TEXT,
+    "AnchorPhotos" TEXT,
+    "DateCompleted" TIMESTAMP WITHOUT TIME ZONE,
+    "CleanupDate" TIMESTAMP WITHOUT TIME ZONE,
+    "CompositeID" TEXT,
+    "Stage" TEXT
+    )"""
+    engine.connect().execute(create_table_statement)
+
+    # Deleting old records
+    object_id_list = ",".join(list(df["CompositeID"].astype(str)))
+    engine.connect().execute(
+        'DELETE FROM "public-health".code55s WHERE "CompositeID" IN (%s)'
```

Comment:
I am seeing an error here:
` operator does not exist: text = integer`
`HINT: No operator matches the given name and argument type(s). You might need to add explicit type casts.`.
Looks like you are casting `object_id_list` to a string above -- could it be safe to remove that cast?

Reply:
I also didn't add quotations around each item in the list, which should be important because they're strings.

Fixing now